src/ensembl/src/shared/radio/Radio.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+
+import defaultStyles from './Radio.scss';
+
+export type radioOption = {
+  value: string;
```

Comment:
Can the value be a number? Can it be a boolean?

Reply:
I thought it should always be a string?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#Value

Reply:
I am almost certain that when you create an input element in JSX, you can pass a number as its value, and it will work (i.e. it will be automatically stringified). Which means that `e.target.value` in the onChange handler will be a string; but if instead of `e.target.value` you pass the original RadioOption value to the `onChange` prop (by finding the appropriate radio option by its index), you can have the radio elements work not only with strings.

Reply:
Cool, I've updated it to support number and boolean.